### PR TITLE
Updated to Spark2.3. Breaking change. 

### PR DESCRIPTION
**Pull request by tomasatdatabricks**

### CODE REVIEW

**Comment by sueann** [comment]
The travis file's spark version probably needs to be updated.

File: src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala [hunk 1]
```diff
@@ -16,9 +16,10 @@
 package org.apache.spark.sql.sparkdl_stubs
 
 import java.util.ArrayList
-import scala.collection.JavaConverters._
 
+import scala.collection.JavaConverters._
 import org.apache.spark.internal.Logging
```
Comment (sueann) [inline src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala]: space above

File: src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala [hunk 2]
```diff
@@ -16,9 +16,10 @@
...
+import org.apache.spark.sql.catalyst.FunctionIdentifier
```
Comment (sueann) [inline src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala]: should be below the next line. did eclipse put these in this ordering?

Comment (tomasatdatabricks) [reply src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala]: Yeah Idea did. I though I set it up with the guidelines.

**Comment by tomasatdatabricks** [comment]
Thanks @sueann. And sorry about the missing travis update, I've created the PR from scratch and rushed it a bit :/

Should be good now. One thing though I get the spark 2.3 from different URL altogether, the one used before doe snot have Spark 2.3

**Comment by thunterdb** [comment]
Fine to me.


On March 14, 2018 at 9:45:53 PM, Sue Ann Hong (notifications@github.com)
wrote:
> sueann commented on this pull request.
>
> this looks good to me. it's fine to merge right? thanks!
>
>
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
>
https://github.com/databricks/spark-deep-learning/pull/102#pullrequestreview-104010640

**Comment by tomasatdatabricks** [comment]
Thanks. I'll merge it then and make a PR for the tf1.6 update.

**Pull request merged**